scripts/camViewer

```diff
@@ -258,15 +274,44 @@ fi
 
 if [ $ENABLE -gt 0 ]; then 
     echo "Enabling ${CAMNAME}"
-    IOC=$(iocfpv "${CAMPVFULL}")
-    $imgr "$IOC" --hutch "$hutch" --enable   
+    for ioc in `iocfpv "$CAMPVFULL:IMAGE1"`; do
+        
+        INFO=$(grep_ioc "$ioc" all | grep "id:'$ioc'")
+        if [ -z "$INFO" ]; then
+             
+            >&2
+        else
+            
+            IOC=$(echo "$INFO" | sed -n "s/^.*id: '\(\S*\)'.*$/\1/p")
+            IOCHOST=$(echo "$INFO" | sed -n "s/^.*host: '\(\S*\)'.*$/\1/p")
+
+            iochutch=$(echo $INFO | grep "/reg/g/pcds/pyps/config/*" | cut -d'/' -f7)
+
+        fi
+    done
+
+    imgr "$ioc" --hutch "$iochutch" --enable   
     exit
 fi
 
 if [ $DISABLE -gt 0 ]; then
     echo "Disabling ${CAMNAME}"
-    IOC=$(iocfpv "${CAMPVFULL}")
-    $imgr "$IOC" --hutch "$hutch" --disable    
+    for ioc in `iocfpv "$CAMPVFULL:IMAGE1"`; do
+
+        INFO=$(grep_ioc "$ioc" all | grep "id:'$ioc'")
+        if [ -z "$INFO" ]; then
+
+            >&2 
+        else
+
+            IOC=$(echo "$INFO" | sed -n "s/^.*id: '\(\S*\)'.*$/\1/p")
+            IOCHOST=$(echo "$INFO" | sed -n "s/^.*host: '\(\S*\)'.*$/\1/p")
+            iochutch=$(echo $INFO | grep "/reg/g/pcds/pyps/config/*" | cut -d'/' -f7)
```

Comment:
Is the grep in this line necessary?

Reply:
I'm not sure how else I would search for that path within $INFO.

Reply:
Doesn't every line in INFO have that string in it?

Reply:
There's only one instance of it. What do you mean?